canvas_modules/common-canvas/src/common-properties/controls/toggle/toggle.jsx

```diff
@@ -54,10 +55,8 @@ class ToggleControl extends React.Component {
 			labelB={labelOn}
 			labelA={labelOff}
 			onToggle={this.handleChange.bind(this)}
-			labelText={this.props.controlItem}
+			labelText={tableLabelText}
```

Comment:
`labelText` is not of type boolean
<img width="1053" alt="image" src="https://user-images.githubusercontent.com/1628100/195418535-9a56485f-ddab-4eb7-95b0-b545729d8b9d.png">

Can you try not setting `labelText` at all if no label exists? 
For example code, you can take a look at how validationProps is being set https://github.com/elyra-ai/canvas/blob/main/canvas_modules/common-canvas/src/common-properties/controls/textfield/textfield.jsx#L97